.github/scripts/patches/qandroidassetsfileenginehandler_dot_slash.diff

```diff
@@ -0,0 +1,34 @@
+--- a/src/plugins/platforms/android/qandroidassetsfileenginehandler.cpp	2023-04-26 08:39:21.720887254 +0200
++++ b/src/plugins/platforms/android/qandroidassetsfileenginehandler.cpp	2023-04-26 08:39:24.477553939 +0200
+@@ -55,6 +55,7 @@
+     if (file.startsWith(assetsPrefix))
+         file.remove(0, prefixSize);
+     file.replace(QLatin1String("//"), QLatin1String("/"));
++    file.replace(QLatin1String("/."), QLatin1String(""));
```

Comment:
These changes would break any files that start with a dot. In lieu of a more accurate normaliser that actually loops over each path fragment to apply "", ".." and ".", each of these sections will need to do `if (file.startsWith("./")) file.remove(0, 2);`, `if (file.endsWith("/.")) file.chop(2);`, and `file.replace("/./", "/");`.

Reply:
Actually, both `QDir` and `QUrl` have a path normaliser so I am not sure why the Android QPA is doing this half-baked thing since it could access the private APIs (`qt_normalizePathSegments`).

Reply:
Since these are asset paths, no files are gonna start with a dot. But I'll take a look if that normalize function will act properly.

Reply:
Linking fails when attempting to use that function, even though it's linking with the module itself, so I guess it's not an exported function that's usable without also getting into messing with that. I don't particularly wanna do that to solve a nonexistent issue, but it's something to suggest when reporting it to Qt.